### PR TITLE
Microswitch Endstop

### PR DESCRIPTION
**Pull request by randellhodges**
Added a PCB for Omron microswitches (others might work) based on a previous hall effect pcb.

### CODE REVIEW

**Comment by timmit99** [comment]
PCB's look great and i love this idea. My only comment is that you seem to have used the older "MZBot" endstop files and we are working to remove the MZBot name from anything we can as we no longer use that name. My thought would be to do a find/replace on your file names/internals and that should be ok.

**Comment by randellhodges** [comment]
Sure, I'll do some renaming and comment again when complete

**Comment by randellhodges** [comment]
Renamed

**Comment by timmit99** [comment]
@john-- looks good to me on a first pass.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Voron-Hardware
 Cool Hardware designed by the Voron Design group
 
-| Project   |      Overview      |  
+| Project   |      Overview      |
 |----------|---------------|
 | Hall Effect Endstop | Replacement for mechanical endstop switches with an adjustable magnetic endstop PCB |
 | Klipper Expander    | Standalone MCU for use with Klipper with various inputs and outputs                 |
+| Microswitch Endstop | PCB for mechanical endstop siwtches                                                 |
```
Comment (Fulg) [inline README.md:8]: Small typo :)
```suggestion
| Microswitch Endstop | PCB for mechanical endstop switches                                                 |
```

Comment (randellhodges) [reply README.md]: What? You don't pronounce it seee-wtch? :)

Fixed.

**Review by FHeilmann — APPROVED** [review]

**Pull request merged**